ts/IterableInAppManager.ts

```diff
@@ -102,9 +102,6 @@ class IterableInAppManager {
      Iterable.logger.log("InAppManager.getHtmlContentForMessage")
 
     return RNIterableAPI.getHtmlInAppContentForMessage(message.messageId)
-      .then((content: any) => {
-        return IterableHtmlInAppContent.fromDict(content)
-      })
```

Comment:
Wow..  the `.then` part was unnecessary the whole time? 
Have you verified if its properly returning just the HTML part?

Reply:
Yes, but I will verify before merging.